doc/source/code/Utilities.rst

```diff
@@ -8,10 +8,55 @@ Apart from the :doc:`operator <Operators>` classes, `eko` also provides some uti
   -  Implementation of the flavor number scheme and the quark thresholds both for
      the :class:`eko.strong_coupling.StrongCoupling` and the :doc:`operators <../theory/Matching>`
 
+  .. include:: IO-tabs/ThresholdConfig.rst
 
-.. include:: IO-tabs/ThresholdConfig.rst
+  When running in |VFNS| it is important to specify the number of flavors active at each given scale, since the evolution path
+  can be different depending of the chosen setting. This path is determined by :meth:`eko.thresholds.ThresholdsAtlas.path`.
 
+  Let us consider an example to better illustrate how it works.
+  Imagine to have a boundary condition ``q2_ref=64, nf_ref=3`` and would like to evolve your object (|PDF| or :math:`a_s`) to a lower
```

Comment:
seems like 64 is not your reference value - the code says Q2=49 - but your target value

Reply:
Actually I guess the error is not in here, but in the code, because otherwise it would be inconsistent with the paths and the rest of the discussion.

Reply:
yes it was wrong copy and paste